src/Tizen.NUI/src/public/ViewProperty/Shadow.cs

```diff
@@ -70,7 +70,8 @@ public Shadow(Shadow other) : this(other == null ? throw new ArgumentNullExcepti
         /// </summary>
         internal Shadow(PropertyMap propertyMap) : base(propertyMap)
         {
-            Color = noColor;
+            Color?.Dispose();
```

Comment:
Since it is constructor and the Color is a member of the Shadow class, so it does not need to dispose.

Reply:
Fix done